src/devices/Ft4222/FtCommon.cs

```diff
@@ -84,7 +95,7 @@ public static (Version chip, Version dll) GetVersions()
 
 
             Version chip = new Version((int)((ftVersion.ChipVersion >> 24) & 0xFF), (int)((ftVersion.ChipVersion >> 16) & 0xFF), (int)((ftVersion.ChipVersion >> 8) & 0xFF), (int)(ftVersion.ChipVersion & 0xFF));
-            Version dll = new Version((int)((ftVersion.DllVersion >> 24) & 0xFF), (int)((ftVersion.DllVersion >> 16) & 0xFF), (int)((ftVersion.DllVersion >> 8) & 0xFF), (int)(ftVersion.DllVersion & 0xFF));
+            Version dll = new Version((int)((ftVersion.dllVersion >> 24) & 0xFF), (int)((ftVersion.dllVersion >> 16) & 0xFF), (int)((ftVersion.dllVersion >> 8) & 0xFF), (int)(ftVersion.dllVersion & 0xFF));
```

Comment:
is dllVersion an int/uint? If so then I don't think the mask is required since you will not have any more bits on the left anyway

Reply:
Correct, those are uint. Remove the & 0xFF